lib/charms/operator_libs_linux/v0/apt.py

```diff
@@ -476,7 +476,7 @@ def from_apt_cache(
             )
         except CalledProcessError as e:
             raise PackageError(
-                "Could not list packages in apt-cache: {}".format(e.output)
+                "Could not list packages in apt-cache: {}".format(e.stderr)
```

Comment:
The call above already uses `universal_newlines=True` (which is now spelled `text=True`). So this one should already be str.